### ISSUE TITLE
Bug in biobb_pdb_tofasta when logging command

### ISSUE THREAD

**Issue by PabloNA97** [bioexcel/biobb_pdb_tools#1]
It's me again! :)

I tried to use the biobb_pdb_tofasta but it's giving me an error inside the launch method line 81:

```python
        self.cmd = [self.binary_path, ' '.join(instructions), self.stage_io_dict['in']['input_file_path'], '>', self.io_dict['out']['output_file_path']]

        fu.log(self.cmd, self.out_log, self.global_log)  # Here it gives the error
```

I think the fu.log tries to concatenate the log prefix with the variable string in line 393 of file_utils.py:

```python
def log(string: str, local_log: Optional[logging.Logger] = None, global_log: Optional[logging.Logger] = None):
    """Checks if log exists

    Args:
        string (str): Message to log.
        local_log (:obj:`logging.Logger`): local log object.
        global_log (:obj:`logging.Logger`): global log object.

    """
    if local_log:
        local_log.info(string)
    if global_log:
        global_log.info(get_logs_prefix() + string)  # Here I have the error
```

I think the argument string in log() should be a string. But in the case of biobb_pdb_tofasta we are giving a list. 

I'm using biobb_pdb_tools 4.2.0

Thank you!

**Comment by PabloNA97** [comment]
Changing the mentioned line 81 to:
```python
fu.log(' '.join(self.cmd), self.out_log, self.global_log)
```
Is doing the trick for me

**Comment by PauAndrio** [comment]
Thanks Plablo it will be fixed in v5.0
